rt/mlc/TBRCTX.MLC

```diff
@@ -60,15 +61,26 @@ BRCTH_1  DS    0H
 *
          SGR   R2,R2               Count iterations
          SGR   R3,R3               Initialize all 64 bits
-         LG      R3,=FL8'3'    Loop count  #370 DSH REP IIHF WITH LG
+         IIHF    R3,3                 Loop count
 BRCTG_1  DS    0H
          AGHI  R2,1                Count iterations (64-bit counter)
+*
+* CODE ADDED BY DSH TO SPEED UP BRCTG TEST
+*
+         LTR   R3,R3               IS LOW R3 ZERO   
+         BZ    SKIP_RESET          YES, SKIP RESET AND PRINT
+         LA    R3,1                NO, RESET TO 1 FOR LAST ITERATION
+         LA    R1,LblBRCTG         Label for message
+         BRAS  R10,ShowMsg         Show message
```

Comment:
While misleading -- the message starts with "After BRCTG" -- it does show the decrement of the high half of GR3. Leave it in, I guess.

Reply:
ok